controller/userController.js

```diff
@@ -0,0 +1,52 @@
+const { User } = require('../model/userSchema');
+
+const updateUserInfo = async (req, res) => {
+    // Get Crendientals from request body
+    const newFirstName = req.body.first_name;
+    const newLastName = req.body.last_name;
+    const newEmail = req.body.email;
+    const newPassword = req.body.new_pword;
+
+    // User ID
+    const userID = req.body._id;
+
+    // Query user by ID
+    const queryID = { _id: userID };
+
+    // Create an update object
+    const fieldsToUpdate = {};
+
+    // Check which fields are filled, add to fieldsToUpdate object if field is filled
+    if (newFirstName) fieldsToUpdate.first_name = newFirstName;
+    if (newLastName) fieldsToUpdate.last_name = newLastName;
+    if (newEmail) fieldsToUpdate.email = newEmail;
+    if (newPassword) fieldsToUpdate.pword = newPassword;
+
+    // Update the date in  user document
+    fieldsToUpdate.created_at_date = new Date().toDateString();
+
+    // Update the time in user document
+    fieldsToUpdate.created_at_time = new Date().toTimeString();
```

Comment:
@raymond-lee093 Do you think we should add updated_at_date and updated_at_time fields to the user document instead of updating the created date and time? This is probably nit picky for this type of project, but let me know your thoughts.

Reply:
I agree with you on that. I’m a perfectionist so I understand haha. It could also be confusing if we left it the way it is. If all we are doing is updating user info there is no need to change the creation date of the whole document. I can go ahead and make that update! Also your changes to the responses for categories were better so I committed them